### PR TITLE
Event teaser, video teaser, blog teaser, four columns layout

### PR DESCRIPTION
**Pull request by retif**
This PR contains new event, video, and blog teasers

## Steps to test:

- [ ] Open Virtual Y homepage
- [ ] Check new teasers

## Quality checks:

Please check these boxes to confirm this PR covers the following cases:

- Maintaining our upgrade path is essential. Check one or the other:
  - [ ] This PR  provides updates via `hook_update_N` or other means.
  - [ ] No updates are necessary for this change.
- Front end fixes should be tested against all of the Open Y Themes.
  - [ ] Tested against Carnation
  - [ ] Tested against Lily
  - [ ] Tested against Rose
  - [ ] This change does not contain front-end fixes.
- [ ] I have flagged this PR "Needs Review" or pinged the VY devs/QA team in Slack

### CODE REVIEW

**Comment by retif** [comment]
retest

**Comment by hamrant** [comment]
retest

**Review by NightHunterSV — APPROVED** [review]

File: js/gated-content/src/assets/svg/Date Icon.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="18" height="20" viewBox="0 0 18 20">
```
Comment (hamrant) [inline js/gated-content/src/assets/svg/Date Icon.svg:1]: Please remove whitespaces from file names `Date Icon.svg ` let's use `date-icon.svg ` instead. Same for the rest of the files

**Review by hamrant — CHANGES_REQUESTED** [review]

**Comment by NadyaRudenko** [comment]
Overall, looks really good. Has some style related issues
1. Left, right and top borders for the live stream, virtual meeting and blog tiles are more light colored in design
2. Time color should be #000000
3. As Kylie noted in Basecamp: 'If more than 2 days out, it will show a day count. Under 2 days, minutes and seconds.' Now it shows days under 2 days, like on screenshot it is 1 day, 18 hours
image

Also, we need to be consistent about the heart icon view. It will be clarified with the client.

**Comment by NadyaRudenko** [comment]
contdown for live streams, virtual meetings for under 2 days should be in format 48:00:00
heart icon should be full on all tiles

**Comment by anpolimus** [comment]
Remaining:
Dashboard is absent
View all = More

**Review by NightHunterSV — APPROVED** [review]

**Review by anpolimus — APPROVED** [review]

File: js/gated-content/src/scss/_fonts.scss [hunk 2]
```diff
@@ -0,0 +1,2 @@
+$cachet-book: "Cachet Book", "Cachet W01 Book", "Cachet", Verdana, sans-serif;
+$cachet-medium: "Cachet Medium", "Cachet W01 Medium","Cachet" Verdana, sans-serif;
```
Comment (NightHunterSV) [inline js/gated-content/src/scss/_fonts.scss:2]: Haven't you missed the comma between "Cachet" and Verdana?

Comment (retif) [reply js/gated-content/src/scss/_fonts.scss]: Yes, thanks! Fixed

**Review by hamrant — APPROVED** [review]

**Comment by NadyaRudenko** [comment]
@retif can you please add more padding to the bottom of the blog tile? so we can avoid the situation when icon is going out of the tile. Like in design, it is under the title. thank you!
image

**Review by NadyaRudenko — APPROVED** [review]
retested, all fixed. thnak you, @retif 🙂

**Pull request merged**